pkg/build/registry/buildlog/rest.go

```diff
@@ -340,38 +313,17 @@ func (r *REST) New() runtime.Object {
 // pipeLogs retrieves the logs for a particular container and streams them into the provided writer.
 func (r *REST) pipeLogs(ctx apirequest.Context, namespace, buildPodName string, containerLogOpts *kapi.PodLogOptions, writer io.Writer) error {
 	glog.V(4).Infof("pulling build pod logs for %s/%s, container %s", namespace, buildPodName, containerLogOpts.Container)
-	location, transport, err := pod.LogLocation(r.PodGetter, r.ConnectionInfo, ctx, buildPodName, containerLogOpts)
```

Comment:
@bparees I didn't find any unit tests for this.  That's not like you.  Are they in another package? I made a mock for the easy path (which works), but not this path and nothing failed....

Reply:
a unit test for what specifically?

mostly the build logs flows are exercised by the extended build tests.

Reply:
> a unit test for what specifically?

For the "stitch multiple log streams into a single log stream" flow.

Reply:
yeah i relied on our extended tests which are all highly dependent on build log output checking.